lib/sacastats_web/templates/character/session.html.heex

```diff
@@ -1,4 +1,47 @@
-<h2><%= @session.name %></h2>
-<h4>Kills: <%= @session.kills %></h4>
-<h4>Deaths: <%= @session.deaths %></h4>
-<h4>KDR: <%= @session.kills / (@session.deaths == 0 && 1 || @session.deaths) %></h4>
+<%= link "<- Back to Sessions", to: Routes.character_path(@conn, :character_sessions, @session.name) %>
+
+<div class="poll-item">
+  Session Summary
+  <ul>
+    <li>Kills: <%= @session.kill_count %></li>
+    <li>Deaths: <%= @session.death_count %></li>
+    <li>KDR: <%= SacaStats.Utils.safe_divide(@session.kill_count, @session.death_count, 2) %></li>
+    <li>Headshot Kills: <%= @session.kill_hs_count %></li>
+    <li>Time Played: 
+      <%= SacaStatsWeb.CharacterView.prettify_duration(@session.login.timestamp, @session.logout.timestamp) %>
+    </li>
+    <li>XP Earned: <%= @session.xp_earned %></li>
```

Comment:
If you are expecting high integers (like > 1000) you can surround that number with a tag that has the `number` class and the js formatter will auto add commas.  Note this will not work properly with decimal numbers. In this case you can change 
`<li>XP Earned: <%= @session.xp_earned %></li>`
to
`<li class="number">XP Earned: <%= @session.xp_earned %></li>`